modules/creating-serverless-apps-yaml.adoc

```diff
@@ -6,10 +6,14 @@
 [id="creating-serverless-apps-yaml_{context}"]
 = Creating serverless applications using YAML
 
-To create a serverless application by using YAML, you must create a YAML file that defines a `Service` object, then apply it by using `oc apply`.
+Creating Knative resources by using YAML files uses a declarative API, which enables you to describe applications declaratively and in a reproducible manner. To create a serverless application by using YAML, you must create a YAML file that defines a Knative `Service` object, then apply it by using `oc apply`.
+
+After the service is created and the application is deployed, Knative creates an immutable revision for this version of the application. Knative also performs network programming to create a route, ingress, service, and load balancer for your application and automatically scales your pods up and down based on traffic.
```

Comment:
"After the service object is. . . " ?

Reply:
I think it's OK as just service here, according to the guidelines we can use either "service" or "`Service` object" as appropriate, and I think that service reads better here.